lib/_layouts/_article.njk

```diff
@@ -9,16 +9,17 @@
 
 {% block header %}
   {{ appArticleHeader({
-    homeHref: homeUrl,
-    "service": {
-      "name": productName,
-      "href": homeUrl
-    },
-    "showNav": "false",
-    "showSearch": "false",
+      homeHref: homeUrl,
+      "showNav": "false",
+      "showSearch": "false",
+      "organisation": {
+        "name": "Business Services Authority",
```

Comment:
at least for the split

Reply:
Hope I've understood correctly - I've added it as a variable (productShortName) to `_base.njk`, also re-named the other one so that it makes sense alongside it (productLongName).

Whilst I was in the code I also thought I'd re-name the overall playbook name from 'Digital Playbook' to 'DDaT Playbook' - hope that's ok?